omniduct/_version.py

```diff
@@ -1,11 +1,13 @@
 import os
 import sys
 
-__all__ = ['__author__', '__author_email__', '__version__']
+__all__ = ['__author__', '__author_email__', '__version__', '__logo__', '__docs_url__']
 
 __author__ = "Matthew Wardrop, Dan Frank"
-__author_email__ = "matthew.wardrop@airbnb.com, dan.frank@airbnb.com"
+__author_email__ = "mpwardrop@gmail.com, dan.frank@airbnb.com"
```

Comment:
@danfrankj You might want your email changed here too.

Reply:
ah yes - would you put danfrankj@gmail.com here ?

Reply:
Done